notebooks_tsqr/NightLog.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "0",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    nr_rep.time_log_as_markdown(nr_adapter, nr_url)
</pre><p>why isn't this day_table()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>